pyaedt/edb_core/hfss.py

```diff
@@ -1024,6 +1024,84 @@ def create_lumped_port_on_net(
                 return edges_pts
         return True
 
+    @pyaedt_function_handler()
+    def create_vertical_circuit_port_on_clipped_traces(self, nets=None, reference_net=None, user_defined_extent=None):
+        """Create an edge port on clipped signal traces.
+
+        Parameters
+        ----------
+        nets : list, optional
+            String of one net or EDB net or a list of multiple nets or EDB nets.
+
+        reference_net : str, Edb net.
+             Name or EDB reference net.
+
+        user_defined_extent : [x, y], EDB PolygonData
+            Use this point list or PolygonData object to check if ports are at this polygon border.
+
+        Returns
+        -------
+        [[str]]
+            Nested list of str, with net name as first value, X value for point at border, Y value for point at border,
```

Comment:
```suggestion
            Nested list of string, with net name as first value, X value for point at border, Y value for point at border,
```
The "str" format should only be used in the first line, which provides the data type. The description itself should use words (and articles as per my previous comment that is now resolved).